utils/proxy/forwarder.py

```diff
@@ -173,16 +173,16 @@ def response(self, flow):
             conn.close()
 
     def _modify_response(self, flow):
-        if self.state.get("mock_remote_config_backend") == "FEATURES":
-            self._modify_response_rc(flow, RC_MOCKED_RESPONSES_FEATURES)
+        if self.state.get("mock_remote_config_backend") == "ASM_FEATURES":
+            self._modify_response_rc(flow, RC_MOCKED_RESPONSES_ASM_FEATURES)
         elif self.state.get("mock_remote_config_backend") == "LIVE_DEBUGGING":
             self._modify_response_rc(flow, RC_MOCKED_RESPONSES_LIVE_DEBUGGING)
         elif self.state.get("mock_remote_config_backend") == "ASM_DD":
             self._modify_response_rc(flow, RC_MOCKED_RESPONSES_ASM_DD)
         elif self.state.get("mock_remote_config_backend") == "ASM_DATA":
             self._modify_response_rc(flow, RC_MOCKED_RESPONSES_ASM_DATA)
-        if self.state.get("mock_remote_config_backend") == "FEATURES_NO_CACHE":
-            self._modify_response_rc(flow, RC_MOCKED_RESPONSES_FEATURES_NO_CACHE)
+        if self.state.get("mock_remote_config_backend") == "ASM_FEATURES_NO_CACHE":
```

Comment:
missing else if no ?
```suggestion
        elif self.state.get("mock_remote_config_backend") == "ASM_FEATURES_NO_CACHE":
```

Reply:
@cbeauchesne

Reply:
Ah, yes. No need to unconditionally always check this 😁 .